scripts/rebuild-testbench.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright(c) 2018 Intel Corporation. All rights reserved.
+
+# fail on any errors
+set -e
+
+print_usage()
+{
+    cat <<EOFUSAGE
+usage: $0 [-f]
+       -f Build testbench with compiler provided by fuzzer
+          (default path: $HOME/sof/work/AFL/afl-gcc)
+EOFUSAGE
+}
+
+rebuild_testbench()
+{
+    cd "$BUILD_TESTBENCH_DIR"
+
+    rm -rf build_testbench
+
+    mkdir build_testbench
+    cd build_testbench
+
+    cmake -DCMAKE_INSTALL_PREFIX=install \
+	  -DCMAKE_VERBOSE_MAKEFILE=ON \
```

Comment:
Would anyone mind `-DCMAKE_BUILD_TYPE=Debug`?

Reply:
for fuzzing i would be against, it would slow fuzzing down. only for user builds i think would be fine